### PR TITLE
Add button to activate flashlight during QR code scanning

### PR DESCRIPTION
**Pull request by aussedatlo**
# Description

Relates to issue: **ZEUS-1109**

Add button in QRCodeScanner to be able to toggle the flashlight.
Sometimes, we need more light to correctly scan a QR code.

https://user-images.githubusercontent.com/35492518/200936661-9117679a-4154-4c7d-afb2-47ac8936e2e5.mp4

This pull request is categorized as a:

- [x] New feature
- [ ] Bug fix
- [x] Code refactor
- [ ] Configuration change
- [ ] Locales update
- [ ] Quality assurance 
- [ ] Other

## Checklist
- [x] I’ve run `npm run tsc` and made sure my code compiles correctly
- [x] I’ve run `npm run lint` and made sure my code didn’t contain any problematic patterns
- [x] I’ve run `npm run prettier` and made sure my code is formatted correctly
- [x] I’ve run `npm run test` and made sure all of the tests pass

## Testing

If you modified or added a utility file, did you add new unit tests?

- [ ] No, I’m a fool
- [ ] Yes
- [x] N/A

I have tested this PR on the following platforms (please specify OS version and phone model/VM):

- [x] Android
- [ ] iOS

I have tested this PR with the following types of nodes (please specify node version and API version where appropriate):

- [x] LND
- [x] c-lightning-REST
- [ ] spark
- [ ] Eclair
- [ ] LndHub

### Locales
- [ ] I’ve added new locale text that requires translations
- [ ] I’m aware that new translations should be made on the Zeus [Transfix page](https://www.transifex.com/ZeusLN/zeus/) and not directly to this repo

### Third Party Dependencies and Packages

- [ ] Contributors will need to run `npm install` after this PR is merged in
- [ ] 3rd party dependencies have been modified:
    * verify that `package.json` and `package-lock.json` have been properly updated
    * verify that dependencies are installed for both iOS and Android platforms

### Other:

- [ ] Changes were made that require an update to the README
- [ ] Changes were made that require an update to onboarding

### CODE REVIEW

File: components/QRCodeScanner.tsx [hunk 1]
```diff
@@ -21,19 +23,37 @@ interface QRState {
 }
 
 export default class QRCodeScanner extends React.Component<QRProps, QRState> {
-    constructor() {
-        super();
+    private maskLength: number;
```
Comment (kaloudis) [inline components/QRCodeScanner.tsx:26]: is `maskLength` used at all?

**Comment by kaloudis** [comment]
Great contribution! Looks good minus the one unused property (commented on above)

**Comment by aussedatlo** [comment]
I've improved the `maskLength` declaration and the typing constructor props

**Comment by kaloudis** [comment]
tACK. Nice job!

**Review by kaloudis — APPROVED** [review]

**Pull request merged**